integration-tests/smoke/auto_ocr_test.go

```diff
@@ -168,6 +222,470 @@ var _ = Describe("Automation OCR Suite @keeper", func() {
 			}, "1m", "1s").Should(Succeed())
 		}
 
+		if testToRun == PauseUnpauseUpkeepTest {
```

Comment:
Why you are breaking all of these out into a table driven test when you are individually breaking them out like this. Would it work better to not use a table and instead use the ginkgo "It" (https://onsi.github.io/ginkgo/#spec-subjects-it) with proper BeforeEach/AfterEach for common setup/cleanup (https://onsi.github.io/ginkgo/#extracting-common-setup-beforeeach https://onsi.github.io/ginkgo/#spec-cleanup-aftereach-and-defercleanup)

Reply:
Just a note - you might need to run the entries in `Ordered` if you are using `BeforeEach/AfterEach`

Reply:
I think we will look at this refactor in a separate ticket as I am not familiar and on my quick first attempt it did not work